enhancements/console/quick-starts.md

```diff
@@ -151,6 +154,23 @@ resource actions. In order for the user to see the Quick Start and
 take its tour, his access review needs to pass all the listed
 resources actions.
 
+#### Quick Start Internationalization
+
+OpenShift can be toggled between multiple languages. The
+Quick Start CRD `spec` contains a `language` field to specify what
+language the Quick Start is in. OpenShift can display the correct
+Quick Starts for each language by comparing the value of the
+`language` field to the currently displayed language in OpenShift.
```

Comment:
What happens if a QuickStart specifies a language that the rest of the UI doesn't support, but is set as the user's language in the system or browser settings? Should we try to handle that case?

Reply:
Good point, should console maybe have a list of supported languages?

Reply:
With the approach outlined here, they would only see English Quick Starts. i18next.languages vs. supported languages will output 'en' if your browser is set to German, for example.

Is it something that third-party Quick Start contributors may want? How often do you think it would come up? Making the Quick Start language match OpenShift creates consistency, but you lose the ability to add pockets of different-language content. I can see the value of both approaches (consistent user experience vs. user experience augmented for your language).